static/js/admin.js

```diff
@@ -1,65 +1,71 @@
 var assignment_release_states = null;
 
 function gradeIndividualItem() {
-    var select3 = document.getElementById("gradingoption3");
-    var colType = select3.options[select3.selectedIndex].value;
-
-    var col1 = document.getElementById("gradingoption1");
-    var col1val = col1.options[col1.selectedIndex].value;
-
-    var col2 = document.getElementById("gradingoption2");
-    var col2val = col2.options[col2.selectedIndex].value;
+    //This function figures out the parameters to feed to getRightSideGradingDiv, which does most of the work
+    var SEL1 = document.getElementById("gradingoption1");
```

Comment:
Is there significance to the all caps variable names?

Reply:
Only to make sure I didn't confuse them with the old variables. If this goes against style I can change them to lowercase.

Reply:
Makes sense, and although I haven't enforced too much in the way of style, it should be easy enough to make them lowercase now.

I plan to set out some style guidelines and variables should definitely be lowercase.

Thanks!

Reply:
The latest commit fixes this.